web/src/app/registration/registration.component.html

```diff
@@ -1 +1,17 @@
-<p>registration works!</p>
+<div id="message"></div>
+<form action="">
+    <label for="firstname">First name:</label>
+    <input type="text" name="firstname" id="firstname">
+    <br>
+    <label for="lastname">Last name:</label>
+    <input type="text" name="lastname" id="lastname">
+    <br>
+    <label for="email">Email:</label>
+    <input type="email" name="email" id="email">
+    <br>
+    <label for="password">Password:</label>
+    <input type="password" name="password" id="password">
+    <br>
+    <input id="register" type="button" value="Register">
```

Comment:
Better to put `<button type="submit">Register</button>` instead of another `input` for button, also simplifies SCSS.